samples/app-cache-meetings/nodejs/assets/sample.json

```diff
@@ -9,7 +9,7 @@
       "This sample feature shows how to improve the subsequent loading time of an App that the user has visited during the current meeting. The tab is loaded as an sidepanel in meeting."
```

Comment:
Is this .zip is correct?
samples/app-cache-meetings/nodejs/**appcachepersonalTab.zip**

It has meeting tab cache as well?

Reply:
actually i was unable to push code to github thir was some problem with network,  so i have created a zip file to share with someone in the team for PR creation.
Yes we can add tab in personal and meeting as well.